src/models/ErrorDetails.ts

```diff
@@ -0,0 +1,10 @@
+interface ErrorDetails {
+    debugMessage: string;
+
+    message: string;
+
+    code?: string;
```

Comment:
Is this optional?

Reply:
It is an optional one but let me know if this one should be removed as well